toplevel/genprintval.ml

```diff
@@ -151,15 +151,15 @@ module Make(O : OBJ)(EVP : EVALPATH with type valu = O.t) = struct
                 (fun x -> Oval_int64 (O.obj x : int64)) ))
     ] : (Path.t * printer) list)
 
-    let exn_printer ppf path =
-      fprintf ppf "<printer %a raised an exception>" Printtyp.path path
+    let exn_printer ppf path _exn =
```

Comment:
Sorry for the nitpick, but I find the use of a variable of the form `_foo` confusing here (and in the rest of the file), given that those variable names are often used for ignored variables, and this variable is very much not ignored. Could you use `exn` instead?